### ISSUE TITLE
Setting time span via config parameter

### ISSUE THREAD

**Issue by yildizmy** [yoshi389111/github-profile-3d-contrib#33]
Would it be possible to set time span via config parameter e.g. 6 months instead of 1 year period? It is also a feature used for similar component and I think it would be useful.

**Comment by yoshi389111** [comment]
What are the advantages of shorter time spans?
Do you want the image to be smaller?

**Comment by yildizmy** [comment]
No, I just want to emphasize a certain period and the image size may be the same, not problem. If it is possible for you, it would let the user flexibility. Thanks in advance.

**Comment by tendil** [comment]
Hello! I can't understand how it is implemented and how can I run it :

***
Launch the added action.

Actions -> GitHub-Profile-3D-Contrib -> Run workflow
***

**Comment by samrocketman** [comment]
It would also be neat to have an "all of time" option to get a full user contribution history.

**Comment by A7med3bdulBaset** [comment]
@yoshi389111 
Are there updates regarding this?

> What are the advantages of shorter time spans? Do you want the image to be smaller?

Contributions look very small, and they take up very little of the image, resulting in a lot of white space.

If I could I would like to limit it to three months.

**Comment by yoshi389111** [comment]
Sorry, but we have abandoned this solution due to the difficulty in balancing the display.
